js/src/setup-mc/setup-stepper/setup-paid-ads/skip-paid-ads-confirmation-modal.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { select } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { noop, merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import AppModal from '.~/components/app-modal';
+import AppButton from '.~/components/app-button';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import { ACTION_SKIP } from './constants';
+import { STORE_KEY } from '.~/data/constants';
+
+/**
+ * Triggered when the skip button is clicked
+ * // TODO: to review
+ *
+ * @event gla_onboarding_complete_button_click
+ */
+
+/**
```

Comment:
Adding tracking doc.

```suggestion

/**
 * @fires gla_documentation_link_click with `{ context: 'skip-paid-ads-modal', link_id: 'paid-ads-with-performance-max-campaigns-learn-more', href: 'https://support.google.com/google-ads/answer/10724817' }`
 */
 
/**
```

Reply:
👍